app/Providers/AppServiceProvider.php

```diff
@@ -64,7 +64,7 @@ public function boot()
                 [
                     ''
                 ],
-                $explode[1]
```

Comment:
This needs to replace the prefixed spaces ( if any ), in the actual base64. So it needs to be index 1